modules/statics/src/coins.ts

```diff
@@ -2,6 +2,8 @@ import {
   account,
   celoToken,
   erc20,
+  erc721,
```

Comment:
I think we can revert changes in `modules/statics/src/coins.ts` since we have taken a different approach